.storybook/main.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     'storybook-css-modules',
     '@storybook/addon-a11y',
     'storybook-addon-rtl',
+    'storybook-dark-mode',
```

Comment:
I went ahead and moved this to dev deps, addressed my previous comments and pushed updates.